### DISCUSSION TITLE
Android Gradle Error

### DISCUSSION THREAD

**Discussion by root19942** [software-mansion/react-native-reanimated#1741] [Q&A]
Hi i have an error due to react-native-reanimated build.gradle  when i try to open android projet after make expo eject

**Comment by jgonet** [comment]
Could you please share more information? Mainly stack trace and package versions (RN, Gradle, React, Reanimated, other relevant).

**Reply by root19942** [reply]
Hi and sorry for the delay
Expo : 4.0.15
react-native-reanimated : "^1.13.2",
react-native-cli: 2.0.1
react-native: 0.63.4
com.android.tools.build:gradle:3.5.3

**Reply by root19942** [reply]
the error on Android Studio

Build file '/Users/neuronx/Desktop/Alaxione/New/Alaxione/node_modules/react-native-reanimated/android/build.gradle' line: 89

A problem occurred configuring project ':react-native-reanimated'.
> Cannot query the value of this provider because it has no value available.

**Comment by root19942** [comment]
someone to help me please
